ethereum/spec/src/test/java/tech/pegasys/teku/spec/logic/common/util/ForkChoiceUtilTest.java

```diff
@@ -182,22 +183,32 @@ public void getSlotStartTime_shouldGetCorrectTimePastGenesis() {
 
   @Test
   void onTick_shouldExitImmediatelyWhenCurrentTimeIsBeforeGenesisTime() {
-    final MutableStore store = mock(MutableStore.class);
+    final MutableStore store = spy(MutableStore.class);
```

Comment:
We can just use a normal mock here.  I'm not a fan of spies if we can possibly avoid them - it can be hard to follow the logic because you often wind up with an odd mix of mocked responses and real ones so never quite know what the test is actually testing or the behaviour it's getting.